packages/browsers/src/CLI.ts

```diff
@@ -135,15 +144,27 @@ export class CLI {
             desc: 'Whether to detach the child process.',
             default: false,
           });
+          yargs.option('system', {
+            type: 'boolean',
+            desc: 'Whether to search for a browser installed on the system instead of the cache folder.',
+            default: false,
+          });
         },
         async argv => {
           const args = argv as unknown as LaunchArgs;
-          const executablePath = computeExecutablePath({
-            browser: args.browser.name,
-            buildId: args.browser.buildId,
-            cacheDir: args.path ?? this.#cachePath,
-            platform: args.platform,
-          });
+          const executablePath = args.system
+            ? computeSystemExecutablePath({
+                browser: args.browser.name,
+                // TODO: validation
```

Comment:
Do we need further validation? We already kind do validation when we go to the switch statement for `platform + channel`.

Reply:
We will need to throw an error if a specific buildId is given or the `latest` which cannot be resolved on the system-wide browsers (updated the todo)